orderbook/src/api.rs

```diff
@@ -130,6 +135,11 @@ pub fn convert_get_trades_error_to_reply(err: anyhowError) -> WithStatus<Json> {
     with_status(internal_error(), StatusCode::INTERNAL_SERVER_ERROR)
 }
 
+pub fn convert_get_presignatures_error_to_reply(err: anyhowError) -> WithStatus<Json> {
+    tracing::error!(?err, "get_presignatures error");
+    with_status(internal_error(), StatusCode::INTERNAL_SERVER_ERROR)
+}
```

Comment:
Why in this file and not get_presignatures.rs ?

Reply:
For consistency with two functions above, `convert_get_orders_error_to_reply` and `convert_get_trades_error_to_reply`. I thought it was done to keep API-generic details like the tracing error and the status code in the same place, but I'd be ok to move all of them to their respective files.

Reply:
Ah I forgot about the other functions. In that case I don't mind either way.